docker/ubuntu_22.04-dev.dockerfile

```diff
@@ -169,45 +118,19 @@ ENV PYNE_DAGMC_ARGS "--dagmc $HOME/opt/dagmc"
 FROM dagmc AS openmc
 ARG build_hdf5
 # build/install OpenMC Python API
-RUN if [ "$build_hdf5" != "NO" ]; then \
-            export HDF5_ROOT="$HDF5_INSTALL_PATH" ; \
-    fi ;\
+RUN export HDF5_ROOT="$HDF5_INSTALL_PATH" ; \
     git clone https://github.com/openmc-dev/openmc.git $HOME/opt/openmc \
```

Comment:
we can clone the specific tag release instead of cloning the repo and checking out the tag

Reply:
```git clone --depth 1 --branch ${openmc_version} https://github.com/openmc-dev/openmc.git $HOME/opt/openmc \```